src/Sulu/Bundle/AdminBundle/Resources/js/components/ArrowMenu/tests/Action.test.js

```diff
@@ -1,43 +1,53 @@
 // @flow
 import React from 'react';
-import {render, mount} from 'enzyme';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import Action from '../Action';
 
 test('Render default Action', () => {
-    expect(render(<Action onClick={jest.fn()}>My Action</Action>)).toMatchSnapshot();
+    const {container} = render(<Action onClick={jest.fn()}>My Action</Action>);
+
+    expect(container).toMatchSnapshot();
 });
 
 test('Render Action with icon', () => {
-    expect(render(<Action icon="su-display-default" onClick={jest.fn()}>My Action</Action>)).toMatchSnapshot();
+    const {container} = render(<Action icon="su-display-default" onClick={jest.fn()}>My Action</Action>);
+
+    expect(container).toMatchSnapshot();
 });
 
 test('Render disabled Action', () => {
-    expect(render(<Action disabled={true} onClick={jest.fn()}>My Action</Action>)).toMatchSnapshot();
+    const {container} = render(<Action disabled={true} onClick={jest.fn()}>My Action</Action>);
+
+    expect(container).toMatchSnapshot();
 });
 
-test('Clicking the Action should call the right handler', () => {
+test('Clicking the Action should call the right handler', async() => {
     const clickHandler = jest.fn();
+    render(<Action onClick={clickHandler}>My Action</Action>);
 
-    const action = mount(<Action onClick={clickHandler}>My Action</Action>);
+    const user = userEvent.setup();
+    await user.click(screen.getByText('My Action'));
```

Comment:
user interactions are asynchronous in `@testing-library/react`. using the `.then()` syntax will be verbose, especially if there are more user interactions in a test case. therefore i decided to use the `await` syntax here (it is used in the `@testing-library/react` documentation too).